products/pubsub/terraform.yaml

```diff
@@ -46,9 +58,22 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       messageStoragePolicy: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
     custom_code: !ruby/object:Provider::Terraform::CustomCode
+      constants: templates/terraform/constants/pubsub_topic.go.erb
       encoder: templates/terraform/encoders/no_send_name.go.erb
       update_encoder: templates/terraform/update_encoder/pubsub_topic.erb
   Subscription: !ruby/object:Overrides::Terraform::ResourceOverride
+    # PubSub resources don't have operations but are eventually consistent.
+    # We add a PollAsync to GET the resource until it exists.
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 6
+      update_minutes: 6
+      delete_minutes: 4
+    async: !ruby/object:Api::PollAsync
+      actions: ['create']
+      operation: !ruby/object:Api::PollAsync::Operation
+        timeouts: !ruby/object:Api::Timeouts
+            insert_minutes: 1
```

Comment:
What's the difference between these timeouts and the resource-level ones?

Reply:
The timeouts are set to the operation/object.async.timeouts unless the resource_override timeouts are given